include/zmq.h

```diff
@@ -682,6 +682,23 @@ ZMQ_EXPORT void zmq_threadclose (void *thread_);
 #define ZMQ_XSUB_VERBOSE_UNSUBSCRIBE 115
 #define ZMQ_TOPICS_COUNT 116
 
+/*  DRAFT NORM socket options                                                 */
+#define ZMQ_NORM_MODE 150
```

Comment:
number these from 117 and up to follow from the rest

Reply:
Yeah, I wasn't sure what to do about that.  I thought it might make sense to separate it numerically to leave room for other potential NORM settings, since there are a lot of NORM configuration options that could still be added.  I suppose in the end, it doesn't really matter though.

Reply:
Done.